ui/sagas/index.js

```diff
@@ -1,9 +1,11 @@
 import { takeLatest } from 'redux-saga/effects';
 
-import { types } from '../ducks/scripts';
+import { FETCH_SCRIPTS } from '../ducks/scripts';
 import { fetchScripts } from './scripts';
+import { FETCH_DEPENDENCIES } from '../ducks/dependencies';
+import { fetchDependencies } from './dependencies';
 
 export default function* index() {
   yield* fetchScripts();
-  takeLatest(types.FETCH_SCRIPTS, fetchScripts);
+  takeLatest(FETCH_SCRIPTS, fetchScripts);
```

Comment:
I wasn't sure where to put `fetchDependencies` here @azz

Reply:
To do them in order:

```js
  yield* fetchDependencies();
```

To do them in parallel:

```js
yield fork(fetchDependencies);
yield fork(fetchScripts);
```